install/ansible/uninstall.sh

```diff
@@ -140,6 +140,9 @@ sed -i.bak "s#__CLUSTER_STORE__#$cluster#g" $env_file
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```

Comment:
if docker version is lesser than 17.03, docker plugin install command is not supported and install fails.